core/src/main/java/org/geysermc/geyser/skin/SkinManager.java

```diff
@@ -277,7 +277,7 @@ public record GameProfileData(String skinUrl, String capeUrl, boolean isAlex) {
             return null;
         }
 
-        private static GameProfileData loadFromJson(String encodedJson) throws IOException, IllegalArgumentException {
+        public static GameProfileData loadFromJson(String encodedJson) throws IOException, IllegalArgumentException {
```

Comment:
You only access loadFromJson from the same package so I think no access modifier would be fine rather than public.

Reply:
sure; lemme fix that